### DISCUSSION TITLE
Ubuntu 22.04 kernel 6.2.0-35

### DISCUSSION THREAD

**Discussion by PatcoreShare** [ipxe/ipxe#1072] [Q&A]
Hi, I know this isn't an iPXE issue, but maybe someone has a solution.
I'm installing Ubuntu 22.04.2 based in part on this guide (https://backreference.org/2013/12/23/diskless-iscsi-boot-with-pxe-howto/index.html). 
The installation goes well, and everything works. After running "apt upgrade," the **kernel changes from 5.19.0-37-generic to 6.2.0-35-generic**. 
Unfortunately, it stops working on 6.2.0, but it works on 5.19.0. Even after running "update-initramfs -u -k all && update-grub.
After booting with the 6.2.0 kernel, it doesn't start up; it hangs during the boot process. In safe mode, it goes to initramfs.
Does anyone have any suggestions on how to resolve this issue or where to look for a solution?

**Comment by NiKiZe** [comment]
Are you booting new kernel from iPXE?
If you are what exactly are you seeing if you are not using quiet mode?

**Reply by PatcoreShare** [reply]
Here it stops and doesn't proceed further.
![ipxeKernel](https://github.com/ipxe/ipxe/assets/11398415/0890a1c9-0de4-4f20-a33a-c017f4e0ebef)
